src/socket_server/server.rs

```diff
@@ -0,0 +1,32 @@
+use crate::socket_server::frame::Frame;
+use crossbeam_channel::Sender;
+use std::io::BufReader;
+use std::os::unix::net::UnixListener;
+use std::thread;
+use dotenv::dotenv;
+use std::env;
+use std::fs::remove_file;
+
+pub fn run_server(tx: Sender<Frame>) {
+    dotenv().ok();
+    let socket_path: String = env::var("ADDRESS").expect("ADDRESS must be set");
+
+    let _ = remove_file(&socket_path);
```

Comment:
Why are you removing this file during [cold start](https://en.wikipedia.org/wiki/Cold_start_(computing))?

Reply:
In case there is any existing socket file in the same path to avoid link errors, just one extra step.